tests/integration-tests/reports/reportsApi-test.js

```diff
@@ -70,6 +72,84 @@ const jobPlatform = process.env.JOB_PLATFORM;
+                    const jobName = 'jobName';
+                    const id = uuid.v4();
+                    const runnerId = uuid.v4();
+                    let chaosExperimentResponse;
+                    const chaosExperimentsInserted = [];
+                    const headersWithContext = Object.assign({}, headers, { 'x-context-id': id });
+
+                    nockK8sRunnerCreation(kubernetesConfig.kubernetesUrl, jobName, id, kubernetesConfig.kubernetesNamespace);
+
+                    for (let i = 0; i < 2; i++) {
+                        const chaosExperiment = chaosExperimentsRequestSender.generateRawChaosExperiment(uuid.v4(), id);
+                        chaosExperimentResponse = await chaosExperimentsRequestSender.createChaosExperiment(chaosExperiment, headersWithContext);
+                        chaosExperimentsInserted.push(chaosExperimentResponse);
+                    }
+
+                    const testCreateResponse = await testsRequestCreator.createTest(basicTest, headersWithContext);
+                    expect(testCreateResponse.status).to.be.equal(201);
+
+                    const testId = testCreateResponse.body.id;
+                    const job = {
+                        test_id: testId,
+                        arrival_rate: 1,
+                        duration: 1,
+                        environment: 'test',
+                        run_immediately: true,
+                        type: 'load_test',
+                        webhooks: [],
+                        emails: [],
+                        experiments: [
+                            {
+                                start_after: 0,
+                                experiment_id: chaosExperimentsInserted[0].body.id,
+                                experiment_name: chaosExperimentsInserted[0].body.name
+                            },
+                            {
+                                start_after: 60000,
+                                experiment_id: chaosExperimentsInserted[1].body.id,
+                                experiment_name: chaosExperimentsInserted[1].body.name
+                            }
+                        ]
+                    };
+
+                    const jobCreateResponse = await jobRequestCreator.createJob(job, headers);
+                    expect(jobCreateResponse.status).to.be.equal(201);
+                    const reportId = jobCreateResponse.body.report_id;
+
+                    const now = Date.now();
+                    const jobExperiment1 = uuid.v4();
+                    const jobExperiment1StartTime = now;
+                    const jobExperiment1EndTime = now + 60000;
+                    await databaseConnector.insertChaosJobExperiment(jobExperiment1, jobCreateResponse.body.id, chaosExperimentsInserted[0].body.id, jobExperiment1StartTime, jobExperiment1EndTime);
```

Comment:
You have to do it yourself directly though db? this is because you don't want to rely on the timing mechanisms?

Reply:
will check it

Reply:
fixed the test - now using nocks over chaos mesh api with full fllow without insertions to db